Sources/Datadog/RUMMonitor.swift

```diff
@@ -287,12 +288,13 @@ public class RUMMonitor {
     /// - Parameters:
     ///   - type: the User Action type
     ///   - attributes: custom attributes to attach to the User Action.
-    public func stopUserAction(type: RUMUserActionType, attributes: [AttributeKey: AttributeValue]? = nil) {
+    public func stopUserAction(type: RUMUserActionType, name: String? = nil, attributes: [AttributeKey: AttributeValue]? = nil) {
```

Comment:
It needs the `public` API comment update with explaining to the user what it is and why/for what it can be `nil`.